tokens/eth/0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19.json

```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "BC",
+  "name": "Block-Chain.com",
+  "type": "ERC20",
+  "address": "0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19",
```

Comment:
```suggestion
  "address": "0x2ecB13A8c458c379c4d9a7259e202De03c8F3D19",
```

Reply:
needs also to be done for the filename - but suggested changes do not yet cover this currently? cc @califa